stor/obs.py

```diff
@@ -272,6 +272,10 @@ def __iter__(self):
         # give users access to underlying buffer)
         return self
 
+    def __del__(self):
+        if hasattr(self, '_buffer') and not self.closed:
```

Comment:
I initially had it in here because I thought it was supposed to be an error to call `close()` twice but I realize that with normal python fileobj you can call close repeatedly so I'll change